src/test/resources/filter.vcf

```diff
@@ -1,11 +1,9 @@
 ##fileformat=VCFv4.1
+##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
```

Comment:
changes to this file are to make it compatible with the HTS schema?

Reply:
to this and all the following files, rather.

Reply:
Yes, exactly.